identity/index.html

```diff
@@ -193,7 +193,10 @@ <h3>MediaDevices.setCaptureHandleConfig()</h3>
             <ul>
               <li>
                 If {{CaptureHandleConfig/handle}} is set to an invalid value, the user agent MUST
-                reject by raising {{TypeError}}.
+                reject by raising {{TypeError}}. Invalid values include:
+                <ul>
+                  <li>Strings whose length exceeds 1024 16-bit characters.</li>
+                </ul>
```

Comment:
Some concerns:
1. This still leaves it unclear what else is "invalid"
2. I see no reason to count storage
3. s/raise/[[=throw=]](https://webidl.spec.whatwg.org/#dfn-throw)/ to be specific
4. The user agent is already required to run these steps, so doesn't need mention in each step

Github won't let me suggest across deleted lines, but I suggest:
```diff
- If {{CaptureHandleConfig/handle}} is set to an invalid value, the user agent MUST
- reject by raising {{TypeError}}.
+ If the length of {{CaptureHandleConfig/handle}} exceeds 1024, [=throw=] {{TypeError}}.
```

I'd also prefer more traditionally numbered "steps" (over "validations") like we have in all other mediacapture specs, as it makes it clear they're sequential.

As it reads now, it's not clear to me exactly what the order of the steps are, and "The user agent MUST" is repeated quite a lot, making it verbose.

Reply:
I think we should open an issue and discuss changes separately. This is an editorial fix that clarifies the original intention in better accordance with spec rules (normative language going where it should). IIUC, your second bullet-point discusses something that should be filed as an issue.

Per-bullet:
1. The PR both includes a definition of invalid values as well as the explanation of what should be done with them.
2. Do you mean you wish to allow arbitrary length? (Please see first paragraph in current comment.)
3. I'll make this change.
4. I don't understand this bullet-point. (Unless you're confused between the <ul> and think it's part of the algorithm, when it is in fact a definition of "invalid" values.)

I am generally fine with your suggestion, given that there is only type of invalidity atm. Assuming that's all you want, I could just do that. Did I get that right?